src/Sulu/Bundle/MediaBundle/Content/Types/MediaSelectionContentType.php

```diff
@@ -168,4 +188,19 @@ public function preResolve(PropertyInterface $property)
             $this->referenceStore->add($id);
         }
     }
+
+    private function getUser()
+    {
+        if (!$this->tokenStorage) {
+            return null;
+        }
+
+        $user = $this->tokenStorage->getToken()->getUser();
+
+        if (\is_object($user)) {
```

Comment:
I've copied that part from somewhere else, so it has already be done this way in other places. Fixed it there too.

Reply:
Maybe it would be better to move the whole `UserInterface` check into the SecurityChecker and remove the `UserInterface` typehints on the other places.